Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/feedback/FeedbackFirstFragment.kt

```diff
@@ -0,0 +1,82 @@
+package org.cornelldti.density.density.facilitydetail.feedback
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.RadioGroup
+import androidx.fragment.app.Fragment
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import androidx.viewpager2.widget.ViewPager2
+import org.cornelldti.density.density.R
+
+
+class FeedbackFirstFragment : Fragment() {
```

Comment:
Should these classes have more descriptive names?

Reply:
yes, this is a good suggestion